src/NzbDrone.Core/MediaFiles/EpisodeImport/Specifications/EpisodeTitleSpecification.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using NLog;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.DecisionEngine;
+using NzbDrone.Core.Download;
+using NzbDrone.Core.Messaging.Commands;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Tv.Commands;
+
+namespace NzbDrone.Core.MediaFiles.EpisodeImport.Specifications
+{
+    public class EpisodeTitleSpecification : IImportDecisionEngineSpecification
+    {
+        private readonly IManageCommandQueue _commandQueueManager;
+        private readonly Logger _logger;
+
+        public EpisodeTitleSpecification(IManageCommandQueue commandQueueManager, Logger logger)
+        {
+            _commandQueueManager = commandQueueManager;
+            _logger = logger;
+        }
+        public Decision IsSatisfiedBy(LocalEpisode localEpisode, DownloadClientItem downloadClientItem)
+        {
+            foreach (var episode in localEpisode.Episodes)
+            {
+                var seriesId = localEpisode.Series.Id;
+                var airDateUtc = episode.AirDateUtc;
+                var title = episode.Title;
+                var episodeNumber = episode.EpisodeNumber;
+
+                if (airDateUtc.HasValue && airDateUtc.Value.Before(DateTime.UtcNow.AddDays(-1)))
+                {
+                    _logger.Debug("Episode aired more than 1 day ago");
+                    continue;
+                }
+
+                if (title.IsNullOrWhiteSpace())
+                {
+                    _logger.Debug("Episode does not have a title and recently aired");
+                    _commandQueueManager.Push(new RefreshSeriesCommand(seriesId));
```

Comment:
I don't think we should refresh, just delay the import.
If you're going to refresh, then you should do it synchronous and check the results again.
If you're adamant about refreshing, then use a single threshold: 
- Define a constant: 1 h
- If we're < 1 h since airdate. Do not refresh. Delay/Reject import.
- If we're >= 1 h since airdate, and lastsync < 1 h since airdate. Refresh, and test again.
- If we're >= 1 h since airdate, and lastsync >= 1 h since airdate. Do not refresh. (so we only forcible refresh once)

That way we only refresh once. I still don't like it, coz it causes all those instances to refresh the same series at the same time. The effect on skyhook could be noticeable.